municipal_finance/compile_data.py

```diff
@@ -270,6 +277,22 @@ def compile_rating_counts(api_url):
 
 
 def compile_data(api_url):
-    compile_profiles(api_url)
-    compile_medians(api_url)
-    compile_rating_counts(api_url)
+    # Setup the client
+    http_client = FuturesSession(
+        executor=ThreadPoolExecutor(max_workers=10)
+    )
+    http_client.mount(
+        "http://",
```

Comment:
not https?

Reply:
Copy and pasted from old code. Changed.

Reply:
if you want this to work both in dev and prod/staging you might need an _http_ and an _https_ mount :)

But perhaps we could just pop api_url un there if it works as a prefix?

Reply:
It seems to be working fine on dev and staging currently, but will definitely fix if it stops working.

Reply:
Apologies. It wasn't working for dev. Commit with fix incoming.